app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(8, 1)) == 8
+    ), "Sum of the parts should be equal to_value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        split_integer(6, 2) == [3, 3]
+    ), "Should split into equal parts when value divisible by parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (
+        split_integer(17, 4) == [4, 4, 4, 5]
+    ), "Should split into 4 parts"
```

Comment:
Your test name isnt's similar to your message. You should've changed test logic after Oleksandrs comment, not message